sources/packages/backend/workflow/BPMN/camunda-8/parttime-assessment-2023-2024.bpmn

```diff
@@ -95,84 +93,212 @@
-    <bpmn:sequenceFlow id="Flow_0mhxp8e" sourceRef="Event_0s8ztj5" targetRef="Event_0bdny14" />
-    <bpmn:intermediateThrowEvent id="Event_105ptqe" name="Calculate Family Income">
+    <bpmn:intermediateThrowEvent id="Event_1e56skq" name="Calculate Total Child Care Costs">
       <bpmn:extensionElements>
         <zeebe:ioMapping>
-          <zeebe:output source="=calculatedDataStudentTotalIncome" target="calculatedDataTotalFamilyIncome" />
+          <zeebe:output source="=if calculatedDataChildCareCost &#62; 0&#10;  then min(((offeringCourseLoad/100) * dmnPartTimeProgramYearMaximums.limitWeeklyChildCare &#10;    * 52 * calculatedDataTotalEligibleDependentsForChildCare),&#10;  calculatedDataChildCareCost)&#10;else 0" target="calculatedDataTotalChildCareCost" />
         </zeebe:ioMapping>
       </bpmn:extensionElements>
-      <bpmn:incoming>Flow_00b4sbw</bpmn:incoming>
-      <bpmn:outgoing>Flow_191hsl9</bpmn:outgoing>
+      <bpmn:incoming>Flow_1m2o8z2</bpmn:incoming>
+      <bpmn:outgoing>Flow_04xt4on</bpmn:outgoing>
     </bpmn:intermediateThrowEvent>
-    <bpmn:intermediateThrowEvent id="Event_0zn13zd" name="Calculate Family Income">
+    <bpmn:intermediateThrowEvent id="Event_0xpdeqq" name="Calculate Total Child Care Costs">
       <bpmn:extensionElements>
         <zeebe:ioMapping>
-          <zeebe:output source="= calculatedDataStudentTotalIncome + calculatedDataPartner1TotalIncome" target="calculatedDataTotalFamilyIncome" />
+          <zeebe:output source="=if calculatedDataChildCareCost &#62; 0&#10;  then min(((offeringCourseLoad/100) * dmnPartTimeProgramYearMaximums.limitWeeklyChildCare &#10;    * 34 * calculatedDataTotalEligibleDependentsForChildCare),&#10;  calculatedDataChildCareCost)&#10;else 0" target="calculatedDataTotalChildCareCost" />
         </zeebe:ioMapping>
       </bpmn:extensionElements>
-      <bpmn:incoming>Flow_189jv8b</bpmn:incoming>
-      <bpmn:outgoing>Flow_0s0yqph</bpmn:outgoing>
+      <bpmn:incoming>Flow_1pdrx9i</bpmn:incoming>
+      <bpmn:outgoing>Flow_173rbte</bpmn:outgoing>
     </bpmn:intermediateThrowEvent>
-    <bpmn:intermediateThrowEvent id="Event_0bdny14" name="calculatedDataFamilySize">
+    <bpmn:exclusiveGateway id="Gateway_1blaf95">
+      <bpmn:incoming>Flow_173rbte</bpmn:incoming>
+      <bpmn:incoming>Flow_04xt4on</bpmn:incoming>
+      <bpmn:incoming>Flow_0wfngj4</bpmn:incoming>
+      <bpmn:outgoing>Flow_19cfg17</bpmn:outgoing>
+    </bpmn:exclusiveGateway>
+    <bpmn:intermediateThrowEvent id="Event_0re3zsg" name="Calculate Child Care Costs">
       <bpmn:extensionElements>
         <zeebe:ioMapping>
-          <zeebe:output source="=calculatedDataTotalEligibleDependants + 1" target="calculatedDataFamilySize" />
+          <zeebe:output source="=if (calculatedDataDependants11YearsOrUnder &#62; 0)&#10;  then studentDataDaycareCosts11YearsOrUnder&#10;else 0" target="calculatedDataDaycareCosts11YearsOrUnder" />
```

Comment:
As a suggestion, the same logic can be simplified using a MAX. Same for the similar ones.

Reply:
I am not able to follow how can we use MAX here as they are 2 different variables.

![image](https://github.com/bcgov/SIMS/assets/54600590/f4bac90a-4db6-4c8c-88d4-b90cfb933d8b)

Let me know if I am missing something here.

Reply:
Never mind, my bad.